InnerEye/ML/Histopathology/notebooks/Explore_PANDA.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
Not entirely sure this fit the package - we don't particularly need to open-source it